### PR TITLE
Replace "nutritious" with "calory dense" to further clarify calory deficit message

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Interface "Replace "nutritious" with "calory dense" to clarify calory deficit message"```

#### Purpose of change
Nutritious might still be too ambiguous,

#### Describe the solution
Replace "nutritious" with "calory dense"

### CODE REVIEW

File: data/json/snippets/health_msgs.json [hunk 1]
```diff
@@ -187,6 +187,6 @@
   {
     "type": "snippet",
     "category": "low_cal",
-    "text": "You feel that your body needs more nutritious food."
+    "text": "You feel that you need to eat more calory dense food."
```
Comment (KorGgenT) [inline data/json/snippets/health_msgs.json]:
```suggestion
    "text": "You feel that you need to eat more calorie dense food."
```

**Comment by Little-119** [comment]
calory

File: data/json/snippets/health_msgs.json (hunk 1 above)
Comment (wapcaplet) [reply data/json/snippets/health_msgs.json]: It should be hyphenated, like "more calorie-dense food."

**Comment by I-am-Erk** [comment]
Maybe a better word would be "sustaining"?

**Comment by Fris0uman** [comment]
Calorie is an information available in the item description, I feel like that's as clear as it gets. It says exactly what the game mechanic expects.

**Pull request merged**